terraform/e2e/main.tf

```diff
@@ -15,6 +15,14 @@
 locals {
   mapping_service_name = "mapping"
   policy_service_name  = "policy"
+
+  alert_subscription_map = {
```

Comment:
If you want to introduce more readability in the e2e module, it is ok that you can local map here. But you should not enforce all users to have a map when using monitoring module.

Reply:
changed to list.